developer_manual/design/navigation.rst

```diff
@@ -2,10 +2,61 @@
 .. codeauthor:: John Molakvoæ <skjnldsv@protonmail.com>
 ..  _navigation:
 
+..  _newbutton:
+
+===============
+Introduction
+===============
+
+The navigation section of any nextcloud app is the left sidebar.
+It is basically composed of
+
+* a 'new' button
+* a menu
+* a settings area
+
+The 'new' button and the settings area are optional.
+
+
 ===============
-App navigation
+New button
 ===============
 
+Introduction
+-------------
+
+A 'new' button is just a stylised button located above the navigation part of your app.
+It can be tied to whatever you want as long as it follow those simple rules.
```

Comment:
to whatever action?

Reply:
Maybe just call it primary action button, since it is then not limited to a create action. And we can  get rid of the `It can be tied to whatever you want` sentence.